README.md

```diff
@@ -260,12 +258,23 @@ sc <- spark_connect(master = "local", version = "2.0.0")
 	```
 	
 4. Note, the `spark_home` parameter in `spark_connect` defaults to the `SPARK_HOME` environment variable. If `SPARK_HOME` is defined it will be always be used unless the `version` parameter is specified to force the use of a locally installed version.
-5. Run the following:
+5. Run the following to create a Spark connection using the default IP and port:
 
 	``` r
 	sc <- spark_connect(master = "local")
 	```
 
+### Spark Connection on Custom Port
```

Comment:
This could be confusing title. I would name this section: "Changing default H2O port"